bubble-sort.js

```diff
@@ -0,0 +1,84 @@
+function criaVetor(maxNumeros) {
+    let vetor = [];
+    
+    for (let i = 1; i < maxNumeros + 1; i++) {
+      vetor.push(i);
+    };
+  
+    return vetor;
+  };
+  
+  function embaralhaVetor(vetorOrdenado) {
+    let vetor = vetorOrdenado;
+    let elementosParaEmbaralhar = vetorOrdenado.length;
+  
+    while (elementosParaEmbaralhar) {
+  
+      let numAleatorio = Math.floor(Math.random() * elementosParaEmbaralhar--); //pega o que falta embaralhar
+      let temp = vetorOrdenado[elementosParaEmbaralhar];
+      vetorOrdenado[elementosParaEmbaralhar] = vetorOrdenado[numAleatorio];
+      vetorOrdenado[numAleatorio] = temp;
+    };
+  
+    return vetor;
+  };
+  
+  function bubbleSort(vetorEmbaralhado) {
+    let vetor = vetorEmbaralhado;
+    
+    for (let vezes = 0; vezes < vetor.length; vezes++) { //percorre todos números do array
+      
```

Comment:
Primeiramente, parabéns! A lógica está certa, mas podemos otimizar algumas partes. Por exemplo: vc faz o primeiro `for` e não utiliza a variável `vezes`, será que podemos usá-la na solução?

Reply:
Fiz uma alteração, agora utilizo essa variável na condição do segundo for.
Não encontrei outra parte que pudesse ser otimizada, aguardo tua análise 🤓 

Mudei os nomes das variáveis pra deixar um pouco mais limpo, de `vezes` para `i` e `posicao` para `j`, tudo bem?